server_test.go

```diff
@@ -970,12 +970,16 @@ func TestServerRoundtripTsig(t *testing.T) {
 
 	s, addrstr, _, err := RunLocalUDPServerWithFinChan(":0", func(srv *Server) {
 		srv.TsigSecret = secret
+		srv.MsgAcceptFunc = func(dh Header) MsgAcceptAction {
+			return MsgAccept
+		}
 	})
 	if err != nil {
 		t.Fatalf("unable to run test server: %v", err)
 	}
 	defer s.Shutdown()
 
+	ch := make(chan struct{})
```

Comment:
I'd prefer something more descriptive than just `ch` considering this isn't integral or obvious. Perhaps something like `receivedCh`.

Reply:
Yeah, I agree. Naming things is hard :-)